docs/administration.rst

```diff
@@ -0,0 +1,82 @@
+Administration
+==============
+
+Several console scripts are provided to facilitate the administration of
+Adhocracy 3.
+
+Workflow state
+--------------
```

Comment:
I would prefer to extend the help that is given on `./bin/set_workflow_state -h` and keeping this section short.